spec/requests/user_controller_spec.rb

```diff
@@ -0,0 +1,26 @@
+require_relative '../rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /index' do
+    before(:example) { get('/users') }
+    it 'http status code should be 200' do
+      expect(response).to have_http_status(200)
+    end
+    it 'renders index template' do
+      expect(response).to render_template('index')
+    end
+    it 'check if user param works properly' do
+      expect(response)
+    end
+  end
+
+  describe 'GET /show' do
+    before(:example) { get('/users/4') }
+    it 'http status code should be 200' do
+      expect(response).to have_http_status(200)
+    end
+    it 'particular users route must exist' do
+      expect(response).to render_template('show')
+    end
+  end
+end
```

Comment:
- [ ] Kindly fix the error.

![image](https://user-images.githubusercontent.com/38635158/180428897-32c3975e-6e59-4af1-9c6c-842f9f7803de.png)